qa/tasks/cephfs/test_fragment.py

```diff
@@ -314,3 +314,78 @@ def _count_fragmented():
             lambda: _count_fragmented() > 0,
             timeout=30
         )
+
+class TestDirfragKillPoints(TestFragmentation, CephFSTestCase):
+    # Two active MDS
+    MDSS_REQUIRED = 2
+
+    init = False
+
+    def _setup_cluster(self):
+        # Set Multi-MDS cluster
+        self.fs.set_max_mds(2)
+
+        self.fs.wait_for_daemons()
+
+        all_daemons = self.fs.get_daemon_names()
+
+        return True
+
+    def _run_dir_frag(self, killpointv):
+
+        status = self.fs.status()
+
+        rank_0 = self.fs.get_rank(status=status, rank = 0)
+        rank_1 = self.fs.get_rank(status=status, rank = 1)
+        rank_0_name = rank_0['name']
+        rank_1_name = rank_1['name']
+
+        command = ["config", "set", "mds_kill_dirfrag_at", str(killpointv)]
+        result = self.fs.rank_asok(command, rank=0, status=status)
+        assert(result["success"])
```

Comment:
This check is unnecessry. If the asok command fials then an exception will be raised.

Reply:
Ack.